BuildLinux.sh

```diff
@@ -80,7 +80,7 @@ fi
 
 DISTRIBUTION=$(awk -F= '/^ID=/ {print $2}' /etc/os-release)
 # treat ubuntu as debian
-if [ "${DISTRIBUTION}" == "ubuntu" ]
+if [ "${DISTRIBUTION}" == "ubuntu" ] || [ "${DISTRIBUTION}" == "linuxmint" ]
```

Comment:
I was going to suggest we future proof this by using ID_LIKE= instead. But it does not seem reliable. 
https://forums.linuxmint.com/viewtopic.php?t=376664